app-marketplace-v2-spec/architecture.md

```diff
@@ -48,8 +53,11 @@
 3. There are 3 kinds of commands:
 - [Slash commands](commands.md#slash-commands) are entered from the message box, with autocomplete.
 - [Embedded commands](commands.md#embedded-commands) are what was previously known as Post Actions or Interactive Messages. They can be embedded in Post Slack attachments, and #STRETCH markdown. **Buttons** launch fully-configured commands, **checkboxes** toggle boolean flags, and **selects** allow to choose a value from a list.
+  - Dani Q: This will include other UI elements, like what I have been working for mobile, right? (channel header button, channel header menu, main menu ...)
 - [Interactive commands](commands.md#interactive-commands) are Interactive Dialogs, or Bot Conversations.
 
+- Dani Q: Should we clarify whether the locations registered are shown to the user during the scope confirmation? In the sense of: "my app register these slash commands: /myapp, /myappanother. It also register these UI elements: 2 post actions (Webapp and mobile), 1 channel header button (webapp), 1 channel menu (mobile), 1 LHS component (webapp). Do you accept?" 
+
```

Comment:
events, and added/modified global UX elements should be pre-declared and enforced.